staging/src/k8s.io/client-go/rest/client_test.go

```diff
@@ -50,12 +49,13 @@ type TestParam struct {
 	testBodyErrorIsNotNil bool
 }
 
-// TestSerializer makes sure that you're always able to decode an unversioned API object
+// TestSerializer makes sure that you're always able to decode metav1.Status
```

Comment:
Some of the changes in this file seem to follow the pattern of the rest of the changes but there's some that don't, why?